src/main/java/org/elasticsearch/action/trainnaivebayes/TrainNaiveBayesResponse.java

```diff
@@ -24,66 +24,44 @@
 import org.elasticsearch.common.io.stream.StreamOutput;
 import org.elasticsearch.common.xcontent.ToXContent;
 import org.elasticsearch.common.xcontent.XContentBuilder;
-import org.elasticsearch.common.xcontent.XContentBuilderString;
 
 import java.io.IOException;
 
 public class TrainNaiveBayesResponse extends ActionResponse implements ToXContent {
 
-    public String getIndex() {
-        return index;
-    }
-
-    public String getType() {
```

Comment:
There is not index anymore elastic/elasticsearch/pull/17650. Scripts are stored in the cluster state now.